AirCasting/CreateSessionViews/ChooseSessionTypeView.swift

```diff
@@ -96,65 +122,44 @@ struct ChooseSessionTypeView: View {
     
     func goToNextFixedSessionStep() {
         createNewSession(isSessionFixed: true)
-        
-        // This will trigger system bluetooth authorisation alert
-        if CBCentralManager.authorization == .notDetermined {
-            _ = bluetoothManager.centralManager
-            didTapFixedSession = true
-        } else if CBCentralManager.authorization == .allowedAlways,
-            bluetoothManager.centralManager.state == .poweredOn {
-            isPowerABLinkActive = true
-        } else {
-            isTurnBluetoothOnLinkActive = true
-        }
     }
     
     var fixedSessionButton: some View {
         Button(action: {
             goToNextFixedSessionStep()
+            if locationTracker.locationGranted == .denied {
```

Comment:
It would be super awesome if this was moved to a simple viewmodel at least as we spoke y-day